src/Components/AdvancedSearch/Internal/QueryBuilder/QueryBuilderRow.tsx

```diff
@@ -332,7 +332,7 @@ const QueryBuilderRow: React.FC<IQueryBuilderRowProps> = (props) => {
             )}
             <div className={classNames.inputColumn}>
                 {/* Property */}
-                <Select<PropertyOption>
+                <Select
```

Comment:
why did we have to drop this typing?

Reply:
This is the type I'm struggling with in tests, it breaks the styles for some reason